Sources/PackageRegistryTool/PackageRegistryTool+Auth.swift

```diff
@@ -88,18 +88,18 @@ extension SwiftPackageRegistryTool {
 
             // compute and validate registry URL
             guard let registryURL = self.registryURL ?? configuration.configuration.defaultRegistry?.url else {
-                throw ConfigurationError.unknownRegistry
+                throw ValidationError.unknownRegistry
```

Comment:
`ValidationError` will print command usage/help, which doesn't seem appropriate here? (same for the other changes in this file)

Reply:
This is not ArgumentParser's `ValidationError` so should be okay?

Reply:
Oh I see. Should we call it something different? Seems a little confusing especially in the context of the implementation of a CLI tool, so both `ValidationError` types will be visible.